website/content/docs/known-issues-and-solutions.md

```diff
@@ -0,0 +1,113 @@
+---
+title: Known Issues and Solutions
+weight: 20
+group: support
+---
+
+## Background
+
+There are a couple of issues that users face while integrating or working with Netlify CMS and while the community is doing their best fixing them, here is a list of known issues and their solutions.
+
+## API Errors
+
+A list of error messages and their possible cause(s) to provide more context while debugging.
+
+#### 1. Error Message.
+
+```
+API_ERROR: {"message":"404 File Not Found"}
+```
+
+#### Possible Causes.
+
+- The collection folder or file property is missing a value (a dedicated file or folder, can't be blank or root). The file or folder should be relative to root (so `src/posts` for example).
+- There are subfolders within the collection folder.
+
+#### 2. Error Message
+
+```
+API_ERROR: Not Found
+```
+
+#### Possible Cause
+
+- The media library folder is missing, or the value of the media library property within config.yml isn't correct.
+
+#### 3. Error Message
+
+```
+Failed to load entries: API_ERROR: Not Found
+```
+
+#### Possible Causes
+
+- Git Gateway is not enabled: Enable Git Gateway from the Netlify Identity settings or refresh the token.
```

Comment:
- Git Gateway is not enabled - I think we show a specific error in that case. If not we should be able to fix the code here https://github.com/netlify/netlify-cms/blob/9119011c8fd304742db7490027de7faf9cec5ee0/packages/netlify-cms-backend-git-gateway/src/implementation.ts#L284. Can you check what is the error reported in that case.
- There aren't any entries for a given collection - don't think this should happen anymore